devel/fb303/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        facebook fb303 1d687797b77b1256b4efe992524ee6cce0fa3c1f
+version             20220112
+checksums           rmd160  ab70801177b36fc825aa7cd1662236a4f3f717cc \
+                    sha256  d60868204b6ce0ed3be39e4e1a69edd022f3c6b0b6d489fb713282f6cdcd0999 \
+                    size    236987
+
+categories          devel
+license             Apache-2
+
+maintainers         nomaintainer
+
+description         fb303 is a base Thrift service and a common set of functionality for querying stats, options, and other information from a service.
+long_description    {*}${description}
+
+github.tarball_from archive
+
+depends_lib-append  port:bison \
+                    port:python38 \
```

Comment:
Must this use python38? python310 is the recommended version now.

Reply:
I think we can safely drop this dependency; it appears to build fine without it.